k8sclient/k8sclient.go

```diff
@@ -335,62 +340,61 @@ func getPodNetworkObj(client *kubernetes.Clientset, netObjs []map[string]interfa
 	return netconf, nil
 }
 
-func getMultusDelegates(delegate string) ([]map[string]interface{}, error) {
-	tmpNetconf := &types.NetConf{}
-	tmpDelegate := "{\"delegates\": " + delegate + "}"
-
+func getMultusDelegates(delegate string) ([]*types.DelegateNetConf, error) {
 	if delegate == "" {
 		return nil, fmt.Errorf("getMultusDelegates: TPR network obj data can't be empty")
 	}
 
-	if err := json.Unmarshal([]byte(tmpDelegate), tmpNetconf); err != nil {
+	n, err := types.LoadNetConf([]byte("{\"delegates\": " + delegate + "}"))
```

Comment:
Twice the LoadNetConf is called in k8sclient.go and also in the multus.go, each time idx 0 is added as the master plugin.  If you have default delegate and 2 delegate from the 2 network objects. Then 0th delegate will be a master plugin from cmdAdd(), 1st delegate also be master plugin because it added here.